bitnami/memcached/templates/statefulset.yaml

```diff
@@ -164,18 +164,28 @@ spec:
           ports:
             - name: {{ .Values.metrics.portName }}
               containerPort: 9150
+          {{- if .Values.livenessProbe.enabled }}
```

Comment:
Wrong variable, it should be `.Values.metrics.livenessProbe.enabled`

Reply:
You are right @sylr , would you so kind to fix it @iamashwanth with another PR please?.